clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -104,6 +105,10 @@ func (pc *PrometheusController) SetUpPrometheusStack() error {
 		}
 	}
 	if err := pc.waitForPrometheusToBeHealthy(); err != nil {
+		klog.Info("Dumping monitoring/prometheus-k8s events...")
+		list, _ := client.ListEvents(k8sClient, namespace, "prometheus-k8s")
```

Comment:
Let's not ignore errors. Here and below.

Reply:
Done.